vnext/ReactUWP/Modules/NativeUIManager.cpp

```diff
@@ -1011,17 +1016,22 @@ void NativeUIManager::UpdateExtraLayout(int64_t tag) {
   if (shadowNode == nullptr)
     return;
 
-  if (shadowNode->IsExternalLayoutDirty()) {
-    YGNodeRef yogaNode = GetYogaNode(tag);
-    if (yogaNode)
-      shadowNode->DoExtraLayoutPrep(yogaNode);
-  }
-
-  for (int64_t child : shadowNode->m_children)
+  for (int64_t child : shadowNode->m_children) {
     UpdateExtraLayout(child);
+  }
 }
 
 void NativeUIManager::DoLayout() {
+  // Process vector of RN controls needing extra layout here.
+  const auto extraLayoutNodes = m_extraLayoutNodes;
+  for (const int64_t tag : extraLayoutNodes) {
+    ShadowNodeBase &node =
+        static_cast<ShadowNodeBase &>(m_host->GetShadowNodeForTag(tag));
+    auto element = node.GetView().try_as<winrt::FrameworkElement>();
```

Comment:
>try_as [](start = 34, length = 6)

nitpick:  You can switch 'try_as' to 'as' (I believe as throws an exception if the cast fails, which it shouldn't).